DeviceComunication/DeviceComunication_CZ.md

```diff
@@ -1048,3 +1092,9 @@ Zařízení vypočítá počet měření, které je potřeba udělat před odesl
 
 Zařízení pak bude čekat mezi měřeními 4 minuty a po každých dvou měření odešle zprávu. Measure
 zprávy se tedy budou odesílat každých 8 minut.
+
+## Zjednodušená implementace potvrzování
+Protože zařízení ve výchozím stavu vyžaduje potvrzení některých zpráv 
+(určuje [5.byte v hlavičce](#5byte---nb-iot)), je nutné implementovat alespoň 
+základní komunikaci ze serveru do zařízení. Na tyto zprávy je možné zjednodušeně 
+odpovídat staticky payloadem `0000000001FF00` který zajistí potvrzení přijaté zprávy.
```

Comment:
Taky mam pocit ze jsem nikde nevidel ten popis magnetu.  Ze se posila event start pri oddaleni a event end pri priblizeni. 

K tomu se vaze jeste informace ze ten prvni event end prijde bez event startu.

Reply:
Je to druhy odstavec u magnetu:

_U magnetického detektoru nastává situace vyplívající z jeho chování, že při prvním použití od restartu (přiblížení magnetu) přijde jako první event zpráva s event typem end._